src/views/metaData/MetaDataDetail.vue

```diff
@@ -40,18 +40,46 @@ import { store } from '../../store.js'
 				<h4>Versie:</h4>
 				<span>{{ metadata.version }}</span>
 			</div>
-			<div>
-				<h4>Properties:</h4>
-				<p>{{ metadata.properties }}</p>
-			</div>
+		</div>
+		<div class="tabContainer">
+			<BTabs content-class="mt-3" justified>
+				<BTab title="Eigenschappen" active>
+					<NcListItem v-for="(value, key, i) in JSON.parse(metadata?.properties)"
+						:key="`${key}${i}`"
+						:name="key"
+						:bold="false"
+						:force-display-actions="true">
+						<template #icon>
+							<ListBoxOutline :class="store.metadataDataKey === key && 'selectedZaakIcon'"
+								disable-menu
+								:size="44"
+								user="janedoe"

```

Comment:
janedoe en jane doe is voorbeeld data die niet in de uiteindenlijke code thuis hoort